### DISCUSSION TITLE
Example Todo List. It won't allow me to continue editing the entry after canceling the discard changes in the warning window.

### DISCUSSION THREAD

**Discussion by emapettinato** [fujidaiti/smooth_sheets#161] [General]
Great job @fujidaiti ! 
The keyboard does not appear to continue editing after Do not discard changes in the warning window when I minimize the modal. 
Thank you

https://github.com/fujidaiti/smooth_sheets/assets/7268433/6b9d34cc-eaa8-426c-b72e-48c06ded3286

**Comment by fujidaiti** [comment]
I think this is a bug in the SDK rather than in this package (see https://github.com/flutter/flutter/issues/148867 and https://github.com/flutter/flutter/issues/149478#issuecomment-2144567424), and it seems that `NavigationSheet` also has the same problem. Hopefully there will be a hotfix soon.

**Reply by emapettinato** [reply]
Thanks for quick response. I will continue the topic in the corresponding place. 
Your work is excellent, I will use it in my next project. Greetings from Argentina.

**Comment by fujidaiti** [comment]
It seems that this problem has been fixed in Flutter 3.22.2. (I have not tested it myself yet.)

>149178 - Fixes an issue on iOS where users are unable to focus on a TextField or open the keyboard again after side pop from another screen.

https://github.com/flutter/flutter/blob/master/docs/releases/Hotfixes-to-the-Stable-Channel.md#3222-june-06-2024

**Comment by fujidaiti** [comment]
Closing this as fixed.
